### ISSUE TITLE
iceprod2 - update core loader script (Trac #891)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#885]
Update loader.sh to point to new cvmfs by default. Try harder to phase out the env tarball, possibly by using parrot?

For standalone systems (for testing usually), possibly put in an assumption about just looking in the base iceprod directory.

Migrated from https://code.icecube.wisc.edu/ticket/891
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:28", 
    "description": "Update loader.sh to point to new cvmfs by default. Try harder to phase out the env tarball, possibly by using parrot?\n\nFor standalone systems (for testing usually), possibly put in an assumption about just looking in the base iceprod directory.", 
    "reporter": "david.schultz", 
    "cc": "ddelventhal", 
    "resolution": "fixed", 
    "_ts": "1550067088921308", 
    "component": "iceprod", 
    "summary": "iceprod2 - update core loader script", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-03-03T18:05:49", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150504T15:58:07: david.schultz changed milestone from "" to "IceProd 2.0"

**Comment by icecube-trac** [comment]
Trac update at 20150811T19:05:42: 
* david.schultz changed status from "new" to "closed"
* david.schultz commented:

> In changeset 136083:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="136083"
> make bin scripts executable. updates to loader.sh to handle testing, new cvmfs, basic parrot support. fixes #885
> ```


* david.schultz changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:28: 
* olivas commented:

> Milestone IceProd 2.0 deleted


* olivas changed milestone from "IceProd 2.0" to ""
